app.js

```diff
@@ -1,8 +1,111 @@
-// import functions and grab DOM elements
+const currentPollSpotEl = document.getElementById('current-poll');
+const pollForm = document.getElementById('poll-form');
+const plusOptionOneButtonEl = document.querySelector('#plus-option-one-button');
+const minusOptionOneButtonEl = document.querySelector('#minus-option-one-button');
+const plusOptionTwoButtonEl = document.querySelector('#plus-option-two-button');
+const minusOptionTwoButtonEl = document.querySelector('#minus-option-two-button');
+const publishPollButtonEl = document.querySelector('#publish-poll-button');
+const pastPollsEl = document.querySelector('#past-polls');
 
-// let state
+let optionOneCounter = 0;
+let optionTwoCounter = 0;
+let pastPollsCounter = 0;
+let currentPoll = {};
+let pastPolls = [];
+let pastPollsStats = [];
```

Comment:
Using a separate array here to track the counters for pastPolls is adding some unnecessary complexity. If you were to build a poll object it would be a nice bundle of all the info needed to be neatly stored in the `pastPolls` array and also for use in the `renderPoll()` function below. Something like:

```
{
  question: "This is a question?",
  optionOne: "First option",
  optionTwo: "Second option",
  optionOneVotes: 5,
  optionTwoVotes: 6
}
```